[12.0] [FIX] purchase_ux & stock_ux: Remove old code.

juanpgarza opened the pull request:
This code is since v9 now it's not necessary and this cause that field qty returned not visible in the tree view of PO.

juanpgarza committed:
[12.0] [FIX] purchase_ux & stock_ux: Remove old code.
This code is since v9 now it's not necessary and this cause that field qty returned not visible in the tree view of PO.

nicomacr commented on purchase_stock_ux/views/purchase_order_views.xml:
```diff
@@ -17,7 +17,7 @@
                 <field name="delivery_status" invisible="1"/>
                 <button name="button_cancel_remaining" type="object" string="Cancel remaining quanitity to be received. This will set line quantity equal to received qty and will cancel remaining moves" icon="fa-ban" confirm="This will set line quantity equal to received qty and will cancel remaining moves. This can not be undone. Are you sure you want to continue?" attrs="{'invisible': [('delive ...
-                <field name="qty_returned" string="Returned Quantity" attrs="{'invisible': [('qty_returned', '=', 0.0)]}" invisible="not context.get('show_purchase')"/>
+                <field name="qty_returned" string="Returned Quantity" attrs="{'column_invisible': [('parent.state', 'not in', ['purchase', 'done'])]}" optional="show"/>
```
the `optional="show"` it's available since v13, remove this part of the code.

nicomacr commented on purchase_ux/__manifest__.py:
```diff
@@ -19,7 +19,7 @@
 {
     'name': 'Purchase UX',
-    'version': '12.0.1.2.0',
+    'version': '12.0.1.4.0',
```
change to `12.0.1.3.0` don't skip a number

nicomacr reviewed (changes requested):
@juanpgarza thanks for the PR some comments below.

juanpgarza committed:
[12.0] [FIX] purchase_ux & stock_ux: Remove old code.
This code is since v9 now it's not necessary and this cause that field qty returned not visible in the tree view of PO.

juanpgarza commented:
@nicomacr thank you for your comments. I already made the corrections
